src/components/AlphanumericInput/AlphanumericInput.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import { chunk } from '../../utils'
+  import { KEY } from '../../const'
+
+  export let value: string
+  export let chunkLength: number
+  let className = ''
+  export { className as class }
```

Comment:
A comment to make it explicit ?

Reply:
Mmmh that's just the way of doing it. Adding always a comment in each component that has a `class` prop is overkill.